### PR TITLE
Manual update (arm and typo)

### PR DESCRIPTION
**Pull request by TApplencourt**

### CODE REVIEW

File: manual/installation.tex [hunk 1]
```diff
@@ -1038,17 +1038,14 @@ \subsection{Installing on systems with ARMv8 based processors}
...
       -DCMAKE_C_FLAGS="-march=armv8-a" -DCMAKE_CXX_FLAGS="-march=armv8-a" \
       -DLAPACK_LIBRARIES="-L$ARMPL_DIR/lib -larmpl_mp" \
       -DFFTW_INCLUDE_DIR="$ARMPL_DIR/include" \
       -DFFTW_LIBRARIES="$ARMPL_DIR/lib/libarmpl_mp.a" \
       ..
 make -j 56
 \end{shade}
-Note that armclang is recognized as an `unknown' compiler by cmake (v3.12.4) and we need to force it as clang to apply necessary flags.
-This workaround will not be necessary in the near future.
+Note that armclang is recognized as an `unknown' compiler by cmake v3.13.* and below. In this case, we need to force it as clang (use (\ishell{-DCMAKE_\{C,CXX\}_COMPILER_ID}) to apply necessary flags.
```
Comment (ye-luo) [inline manual/installation.tex]: Please list the "necessary flags" explicit.

Comment (TApplencourt) [reply manual/installation.tex]: Done

**Comment by prckent** [comment]
@TApplencourt I approved, but does DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=98 actually work? Don't we need 14?

**Comment by ye-luo** [comment]
@prckent I added that line. default is the compiler default if you don't specify --std=c++xx. We ask explicitly for C++14 this is different.

**Comment by ye-luo** [comment]
Okay to test

**Pull request merged**